lib/Utility/BigInteger.php

```diff
@@ -56,12 +56,8 @@ public static function create($val)
             if (extension_loaded('gmp')) {
                 $ret = new BigIntegerGmp();
             }
-            elseif (extension_loaded('bcmath')) {
-                $ret = new BigIntegerBcmath();
-            }
             else {
-                // TODO: potentially offer pure php implementation?
-                throw new \RuntimeException('Requires GMP or bcmath extension.');
+	            $ret = new BigIntegerBcmath();
```

Comment:
I think there's a tab affecting whitespace here, could you update with 4 spaces instead of tab? more visible here https://github.com/fgrosse/PHPASN1/blob/0e09d84161894273577ac7c733877506fd46e323/lib/Utility/BigInteger.php#L60